device-types/Longshine/LCS-FS9116-B.yaml

```diff
@@ -0,0 +1,49 @@
+---
+manufacturer: Longshine
+model: LCS-FS9116-B
+slug: longshine-lcs-fs9116-b
+part_number: GEU-1629
+comments: '[Datasheet](https://cdn-reichelt.de/documents/datenblatt/E910/LCS-FS9116%23LCS.pdf)'
+airflow: left-to-right
+weight: 1.5
+weight_unit: kg
+is_full_depth: false
+front_image: true
+rear_image: true
+u_height: 1
+interfaces:
+  - name: GigabitEthernet1
+    type: 1000base-t
```

Comment:
Hi Thank you for your contribution.
Looking at the spec sheet this device is only 100mb not 1gb therefore all interfaces need to be of 'type: 100base-tx'